packages/firebase_ai/firebase_ai/example/lib/pages/bidi_page.dart

```diff
@@ -258,21 +255,25 @@ class _BidiPageState extends State<BidiPage> {
       _recording = true;
     });
     try {
-      await _audioRecorder.checkPermission();
-      final audioRecordStream = _audioRecorder.startRecordingStream();
+      var inputStream = await audioInput.startRecordingStream();
+      await audioOutput.playStream();
       // Map the Uint8List stream to InlineDataPart stream
-      final mediaChunkStream = audioRecordStream.map((data) {
-        return InlineDataPart('audio/pcm', data);
-      });
-      await _session.sendMediaStream(mediaChunkStream);
+      if (inputStream != null) {
+        Stream<InlineDataPart> inlineDataStream = inputStream.map((data) {
```

Comment:
[nit] Consider omitting the variable type on the left.

```suggestion
        final inlineDataStream = inputStream.map((data) {
```